runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordPlugin.java

```diff
@@ -290,7 +327,8 @@ public void onResponse(Call call, Response response) throws IOException
 					{
 						image = ImageIO.read(inputStream);
 					}
-					memberById.setAvatar(image);
+
+					partyService.setPartyMemberAvatar(memberById.getMemberId(), image);
```

Comment:
Added an event for when the avatar changes. I'd like a review on that. Thanks!

Changes https://github.com/runelite/runelite/pull/13406/commits/822d88d6bfcb9ab0e34c83eab8c7815d87ba8219